### ISSUE TITLE
MorphTo field error when creating via "class name must be valid object or string"

### ISSUE THREAD

**Issue by scottgrayson** [laravel/nova-issues#2921]
- Laravel Version: 7.28.3
- Nova Version: 3.11.1
- PHP Version: 7.4.10
- Database Driver & Version:
- Operating System and Version: macos 10.15.7
- Browser type and version: chrome

### Description:

Morph to field is erring when using the create via functionality. This is not a problem on 3.10.0 it errors when i upgrade to 3.11.1

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. have a Nova desource A that uses MorphTo with one of the types as resource B
2. create a Nova resource B that relates to A using a MorphToMany field
3. view resource B detail page
4. click create A from the relations section of the page
5. see redirected to the create form for A
6. see error

```
exception: "Error"
file: "/Users/scottgrayson/Code/CSC/vendor/laravel/nova/src/Http/Controllers/MorphableController.php"
line: 56
message: "Class name must be a valid object or a string"
```

**Comment by crynobone** [comment]
Able to replicate this.

**Comment by crynobone** [comment]
Fixed via laravel/nova#988
